src/Mod/Draft/draftguitools/gui_selectplane.py

```diff
@@ -198,6 +198,9 @@ def handle(self):
             if hasattr(sel.Object, 'TypeId') and sel.Object.TypeId == 'App::Part':
                 self.setPlaneFromObjPlacement(sel.Object)
                 return True
+            elif hasattr(sel.Object, "TypeId") and sel.Object.TypeId == "PartDesign::Plane":
+                self.setPlaneFromObjPlacement(sel.Object)
+                return True
```

Comment:
Consistency?
Single quote: 50 occurrences.
Double quote: 400+ occurrences.

Reply:
Local consistency. :) At least with the few lines around. ;)

Reply:
The [Draft coding conventions](https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/Draft/coding_conventions.md) only allow double quotes.

Reply:
That's only a comment. Do whatever you want with it. ;)

Reply:
> That's only a comment.

I hope you are not referring to the [Draft coding conventions](https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/Draft/coding_conventions.md) here. On a previous occasion 3 (!) people commented on 2 (!) missing spaces in a single line of code I added. So apparently the coding conventions are not a trivial matter...

Reply:
Honestly these conventions are there just because someone decided to write them... They are not the fruit of a complex brainstorming with all developers involved. Of course we should zeal for styling and coherent way to do things, but IMHO as long as it works and it is clear and understandable by others, everything is fine (just my opinion of course). In french we say "le mieux est l'ennemi du bien" (the better is the enemy of the good) ;)